espefuse/efuse/emulate_efuse_controller_base.py

```diff
@@ -10,12 +10,6 @@
 
 from bitstring import BitStream
 
-try:
-    FileNotFoundError
-except NameError:
-    # Python 2.7 compatibility
-    FileNotFoundError = IOError
-
```

Comment:
Should we drop python2.7 support? I think it is logical, as esp idf uses python3 now and most OSs have python3 installed by default. But if this is the case, wouldn't be better to check if the user is running a compatible python version? Something like this:

```
import sys

if sys.version_info[0] < 3:
    sys.exit("Python 2 is not supported. Please use Python 3.")
```